### ISSUE TITLE
Improve the setup instructions

### ISSUE THREAD

**Issue by gravitystorm** [cyclestreets/cyclescape#99]
https://github.com/cyclestreets/toolkit/wiki/Software-installation is long and overcomplicated, and could easily put people off from contributing.

A few things that stand out in a quick skim-read:
- postgres 9.0 or 9.1 should both work, so notes can be simplified. Recommend a newer distribution if necessary to simplify notes.
- annotate comes with bundle install, no need to install separately
- git setup instructions are unnecessary - just link to github ones. Should only come later when discussing pull requests
- Apache is unnecessary for rails development. Use rails server
- Don't create a new unix user just for rails to use with postgres. For development, use your own user account
- Either create your own account as a superuser, or point out the joys of sudo -u postgres
- Don't set up passwords for postgres - just connect using your own account
- ruby - avoid all the rvm stuff if possible. It just works on ubuntu + ruby1.9
- Use system nodejs, don't bother adding ppas
- Mention lvh.me as an addendum regarding domain names, rather than editing hosts files.

Basically we want to minimise the time, effort and stress for getting from start to editing code.
